metrics/details/init.lua

```diff
@@ -23,7 +23,7 @@ end
 function Registry:is_registered(collector)
     for _, c in ipairs(self.collectors) do
         if c.name == collector.name and c.kind == collector.kind then
-            return true
+            return true, c
```

Comment:
I suggest you to consider an implementation of new method "get" that will return you "collector" if it is registred and nil otherwise.

I think we should keep semantics when "is_..." returns only true or false

Reply:
Done, added separate method to get from cache